db/migrate/20150709065335_create_rockauth_users.rb

```diff
@@ -1,6 +1,8 @@
 class CreateRockauthUsers < ActiveRecord::Migration
   def up
     create_table :users do |t|
+      t.string :first_name
```

Comment:
Since you are adding this to the original migration you will have to write a similar migration for canonical api.